lua/packer/display.lua

```diff
@@ -593,27 +593,33 @@ local function make_filetype_cmds(working_sym, done_sym, error_sym)
   return {
     -- Adapted from https://github.com/kristijanhusak/vim-packager
     'setlocal buftype=nofile bufhidden=wipe nobuflisted nolist noswapfile nowrap nospell nonumber norelativenumber nofoldenable signcolumn=no',
-    'syntax clear', 'syn match packerWorking /^ ' .. working_sym .. '/',
+    'syntax clear',
+    'syn match packerWorking /^ ' .. working_sym .. '/',
     'syn match packerSuccess /^ ' .. done_sym .. '/',
     'syn match packerFail /^ ' .. error_sym .. '/',
-    'syn match packerStatus /\\(^+.*—\\)\\@<=\\s.*$/',
-    'syn match packerStatusSuccess /\\(^ ' .. done_sym .. '.*\\)\\@<=\\s.*$/',
-    'syn match packerStatusFail /\\(^ ' .. error_sym .. '.*\\)\\@<=\\s.*$/',
-    'syn match packerStatusCommit /\\(^\\*.*—\\)\\@<=\\s.*$/',
+    'syn match packerStatus /^+.*—\\zs\\s.*$/',
+    'syn match packerStatusSuccess /\\(^ ' .. done_sym .. '\\)\\@5<=\\s.*$/',
```

Comment:
OOI why specifically 5 for the number of bytes to check here? only just began reading through the docs on this seems good to have the limit just curious about how you arrived at 5?

Reply:
Lowest it worked on was 4 + 1 . Nothing scientific about it lol

Reply:
Could this then recur for users who have changed their `done_sym` and `err_sym` to other bigger characters since a user can configure this? The docs seemed to imply there was a way to do this but limiting the check to the current line or would that still bring on this bug?

Reply:
Maybe we can check len of `done_sym` and `err_sym`?

Reply:
Lua's len function returns the size in bytes so seems like it'd work for this case. Read through the docs and seems the default i.e. what we already had was the one that is limited to the current line and the line above but seems ambiguous about it being slow and was already causing this issue so is best avoided

Reply:
Ok . @akinsho  Now the highlight should work no mater what user has in config :)

Reply:
> Lua's len function returns the size in bytes so seems like it'd work for this case.

I learned that when `string.sub()` managed to split a multibyte char in two :P